In-house patch to remove unnecessary dependencies from Ironic's
requirements files. The specific reasons are as follows:

websockify  Not applicable

oslo.rootwrap Not applicable to Solaris

--- ironic-5.1.0/requirements.txt.~1~	2016-03-31 01:16:58.699655912 -0800
+++ ironic-5.1.0/requirements.txt	2016-03-31 01:17:28.642059472 -0800
@@ -18,12 +18,10 @@ python-swiftclient>=2.2.0 # Apache-2.0
 pytz>=2013.6 # MIT
 stevedore>=1.5.0 # Apache-2.0
 pysendfile>=2.0.0 # MIT
-websockify>=0.6.1 # LGPLv3
 oslo.concurrency>=3.5.0 # Apache-2.0
 oslo.config>=3.7.0 # Apache-2.0
 oslo.context>=0.2.0 # Apache-2.0
 oslo.db>=4.1.0 # Apache-2.0
-oslo.rootwrap>=2.0.0 # Apache-2.0
 oslo.i18n>=2.1.0 # Apache-2.0
 oslo.log>=1.14.0 # Apache-2.0
 oslo.middleware>=3.0.0 # Apache-2.0
